usaspending_api/recipient/delta_models/sam_recipient.py

```diff
@@ -1,29 +1,107 @@
 SAM_RECIPIENT_COLUMNS = {
-    "awardee_or_recipient_uniqu": "STRING",
+    "id": "LONG NOT NULL",
+    "recipient_hash": "STRING",
     "legal_business_name": "STRING",
-    "ultimate_parent_unique_ide": "STRING",
-    "ultimate_parent_legal_enti": "STRING",
-    "broker_duns_id": "INTEGER NOT NULL",
-    "update_date": "DATE NOT NULL",
+    "duns": "STRING",
     "address_line_1": "STRING",
     "address_line_2": "STRING",
+    "business_types_codes": "ARRAY<STRING>",
     "city": "STRING",
     "congressional_district": "STRING",
     "country_code": "STRING",
+    "parent_duns": "STRING",
+    "parent_legal_business_name": "STRING",
     "state": "STRING",
-    "zip": "STRING",
     "zip4": "STRING",
-    "business_types_codes": "ARRAY<STRING>",
-    "dba_name": "STRING",
-    "entity_structure": "STRING",
+    "zip5": "STRING",
+    "alternate_names": "ARRAY<STRING>",
+    "source": "STRING NOT NULL",
+    "update_date": "TIMESTAMP NOT NULL",
     "uei": "STRING",
-    "ultimate_parent_uei": "STRING",
+    "parent_uei": "STRING",
 }
 
-sam_recipient_sql_string = rf"""
+sam_recipient_create_sql_string = fr"""
     CREATE OR REPLACE TABLE {{DESTINATION_TABLE}} (
         {", ".join([f'{key} {val}' for key, val in SAM_RECIPIENT_COLUMNS.items()])}
     )
     USING DELTA
     LOCATION 's3a://{{SPARK_S3_BUCKET}}/{{DELTA_LAKE_S3_PATH}}/{{DESTINATION_DATABASE}}/{{DESTINATION_TABLE}}'
     """
+sam_recipient_load_sql_string = fr"""
+-- Create temp table to reload into the duns table without dropping the destination table
+DROP TABLE IF EXISTS int.sam_recipient;
+
+CREATE TABLE int.sam_recipient AS (
```

Comment:
Here we should try to avoid recreating the table. Check out the pattern of `INSERT OVERWRITE` used by `award_search_load_sql_string` and `transaction_search_load_sql_string`

Reply:
[INSERT OVERWRITE docs](https://docs.databricks.com/sql/language-manual/sql-ref-syntax-dml-insert-into.html)

> If you specify OVERWRITE the following applies:
> - Without a partition_spec the table is truncated before inserting the first row.
> - Otherwise all partitions matching the partition_spec are truncated before inserting the first row.